src/lib.rs

```diff
@@ -34,37 +47,36 @@ impl Pages {
         if page.length > 0 {
             page.end -= 1;
         };
-        page.html = (self.f)(page.begin, page.length);
-        page
+
+        Ok(page)
     }
 
     pub fn offset(&self) -> usize {
-        self.offset
+        self.page_number
     }
```

Comment:
The function name is supposed to be `page_number`

Reply:
It's fixed